packages/persistent-merkle-tree/src/hasher/noble.ts

```diff
@@ -0,0 +1,10 @@
+import {sha256} from "@noble/hashes/sha256";
+import type {Hasher} from "./types";
+import {hashObjectToUint8Array, uint8ArrayToHashObject} from "./util";
+
+const digest64 = (a: Uint8Array, b: Uint8Array): Uint8Array => sha256.create().update(a).update(b).digest();
```

Comment:
see if this helps:

```
const digest64 = (a: Uint8Array, b: Uint8Array): Uint8Array => sha256(concatBytes(a, b))
```

concatBytes can be imported from noble/hashes/utils

Reply:
would it give perf boost, from my understanding the underlying lib (for e.g. as-sha256) will be calling chunkified updates itself if you pass bigger input

Reply:
probably won't, but just in case